core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -188,6 +189,10 @@ private static boolean validateTransportType(TransportType transportType, String
 
     @Nullable
     private static Boolean useOpenSsl;
```

Comment:
`useOpenSsl` could be removed in favor of `TLS_ENGINE_TYPE`.

Reply:
I'll remove it 👍